examples/variables/variables.bal

```diff
@@ -6,12 +6,14 @@ int total = 99;
 // Declare a public module-level variable.
 public int count = 0;
 
-// Declare a constant. `const`'s value is frozen. Value is computed at compile time therefore immutable.
+// Declare a constant. Value of the constant is frozen. Value is computed at
+// compile time therefore is immutable.
```

Comment:
Shall we rephrase this? This feels like disconnected sentence fragments.

Reply:
IMO, we don't need to go into details about constants here. 

Ideally the constants BBE should have details about constants always being frozen (immutable), and here we can just say "Declare a constant."

WDYT?